style.css

```diff
@@ -220,34 +290,342 @@ h2 {
+  transition-property: background-color font-size color;
+  transition-duration: 200ms 500ms 200ms;
 }
 
-/* End of projects section */
+.lng-list div {
+  background-color: #f7f7f9;
+  justify-content: center;
+  align-self: stretch;
+  border-radius: 10px;
+  margin: 0;
+}
 
-@media only screen and (max-width: 600px) {
-  .container {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-content: center;
-    min-height: 100vh;
-    width: 100%;
-    padding: 0;
-    margin: 0;
+.lng li:hover {
+  font-size: 1.2em;
+  background-color: #7a869a;
+  color: #ebebff;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: solid #396df2 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #fff;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
+}
+
+.btn:hover {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: solid #396df2 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #6070ff;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
+}
+
+.btn:active {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: solid #396df2 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #2230d2;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
+}
+
+.btn:disabled {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #5e6c84;
+  border: solid #5e6c84 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #fff;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
+}
+
+/* About section */
+.about {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-self: stretch;
+  background: #fff;
+  border-top-right-radius: 80px;
+  margin-top: 142px;
+}
+
+.about .grid-container {
+  padding: 0 9.86%;
+}
+
+/* Contact form styling */
+.form-wrapper {
+  width: inherit;
+  background-color: #fff;
+}
+
+.form-header {
+  max-width: 570px;
+  text-align: center;
+}
+
+.form-header h2 {
+  margin-top: 110px;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+}
+
+.form-header p {
+  color: #ebebff;
+  margin: 12px 0;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.contact {
+  width: 30%;
+}
+
+.contact ul {
+  padding: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 20px;
+}
+
+.contact ul li input {
+  padding: 10px;
+  color: #172b4d;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+textarea {
+  padding: 10px;
+  color: #172b4d;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+.contact-btn {
+  padding: 12px;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [x] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.contact-btn {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}

.contact-btn:disabled {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: inter, sans-serif;
}
```

> **Note**: Also, you can do the same thing with the `display: flex` declaration.